Dockerfile

```diff
@@ -102,12 +98,30 @@ WORKDIR /src
 ARG TESTPKGS
 RUN --mount=type=cache,target=/root/.cache/go-build --mount=type=cache,target=/go/pkg --mount=type=cache,target=/tmp go test -v -covermode=atomic -coverprofile=coverage.txt -coverpkg=${TESTPKGS} -count 1 ${TESTPKGS}
 
-FROM scratch AS kres-linux-amd64
-COPY --from=kres-linux-amd64-build /kres-linux-amd64 /kres-linux-amd64
+FROM embed-generate AS embed-abbrev-generate
+WORKDIR /src
+RUN echo -n 'undefined' > internal/version/data/sha && \
+    echo -n 'undefined' > internal/version/data/tag
```

Comment:
doesn't it always overwrites version with `undefined` in that case?

Reply:
No - we use `embed-generate` stage to overwrite tag and sha values from `generate` stage. I answered in a bit more details to Noel.